src/utils/data_handling.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+import time
+import pandas as pd
+from core.state import PropagatedOutput
+from dataclasses import asdict
+
+
+def states_to_df(states: List[PropagatedOutput]) -> pd.DataFrame:
+    dataframes: List[pd.DataFrame] = []
+    for output in states:
+        flattened_output_dict = pd.json_normalize(asdict(output))
+        single_dataframe = pd.DataFrame.from_dict(flattened_output_dict)
+        dataframes.append(single_dataframe)
+
+    complete_df = pd.concat(dataframes)
+    return complete_df
+
+
+def df_to_csv(dataframe: pd.DataFrame):
+    dataframe.to_csv(f"runs/cislunarsim-{time.time()}")
```

Comment:
I get this error when I run main.py from the `src/` directory. Running it from `CislunarSim/` results in no errors, so might want to change how you specify this path 
![image](https://user-images.githubusercontent.com/54723402/162842952-e30662ad-a648-4b6a-9e2c-97244fa03548.png)

Reply:
Done